flexget/options.py

```diff
@@ -473,3 +471,89 @@ def parse_args(self, *args, **kwargs):
         # Set the 'allow_manual' flag to True for any usage of the CLI
         setattr(result, 'allow_manual', True)
         return result
+
+
+class CLITableError(Exception):
+    """ A CLI table error"""
+
+
+class CLITable(object):
+    """
+    A data table suited for CLI output, created via its sent parameters.
+    """
+
+    def __init__(self, type, table_data, title=None):
+        self.title = title
+        self.type = type
+        self.table = self.supported_table_types()[type](table_data)
+
+    @property
+    def output(self):
+        self.table.title = self.title
+        if self.type == 'porcelain':
+            # porcelain is a special case of AsciiTable
+            self.table.inner_footing_row_border = False
+            self.table.inner_heading_row_border = False
+            self.table.outer_border = False
+
+        if self.table.ok:
+            return '\n' + self.table.table
+        raise CLITableError(
+            'Terminal size is not suffice to display table. Terminal width is {}, table width is {}. '
+            'Consider setting a lower value using `--max-width` option.'.format(terminal_size()[0],
+                                                                                self.table.table_width))
+
+    @staticmethod
+    def supported_table_types(keys=False):
+        """
+        This method hold the dict for supported table type. Call with `keys=True` to get just the list of keys.
+        """
+        table_types = {
+            'plain': AsciiTable,
+            'porcelain': AsciiTable,
+            'single': SingleTable,
+            'double': DoubleTable,
+            'github': GithubFlavoredMarkdownTable
+        }
+        if keys:
+            return list(table_types)
+        return table_types
+
+    @staticmethod
+    def colorize(text, color_tag, porcelain=False):
+        """
+        This method calls for `colorclass` to try and color the given text
+        :param text: Text to color
+        :param color_tag: Color tag. Should adhere to colorclass.list_tags().
+        :param porcelain: If True, no colors should be returned.
+        :return: Text or colorized text
+        """
+        if porcelain:
+            return text
+        if sys.platform == 'win32':
+            Windows.enable(auto_colors=True)
+        return Color('{%s}%s{%s}' % (color_tag, text, '/' + color_tag))
+
+    @staticmethod
+    def word_wrap(text, max_length=0):
+        """
+        A helper method designed to return a wrapped string. This is a hack until (and if) `terminaltables` will support
+        native word wrap.
+        :param text: Text to wrap
```

Comment:
you need empty line before any

    :param things

otherwise sphinx will not work correctly